code42/py42
# Upgrade device

### Description of Change ###

Adds `sdk.devices.upgrade()` to upgrade an individual device. Currently administrators can do this via the console, adding it to py42 enables them to automate and have more flexibility in terms of upgrading devices in a phased manner instead of all-or-nothing.

### Issues Resolved ###

- closes #394 

### Testing Procedure ###

1. Create a device in the Code42 cloud running an older version of the client (using the delayed client upgrade functionality)
2. Call `sdk.devices.upgrade()` with that device's GUID
3. Verify that the authority received the instruction to upgrade this client

### PR Checklist ###
Did you remember to do the below?
